File: app/src/main/java/ru/artempugachev/popularmovies/model/ReviewResponse.java
package ru.artempugachev.popularmovies.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import ru.artempugachev.popularmovies.model.Review;

/**
 * Data class for review response
 */

public class ReviewResponse {
    @SerializedName("id")
    private String id;

    @SerializedName("page")
    private int page;

    @SerializedName("results")
    private List<Review> results;

    public List<Review> getResults() {
        return results;
    }
}

File: app/src/main/java/ru/artempugachev/popularmovies/model/ReviewResponse.kt
package ru.artempugachev.popularmovies.model

import com.google.gson.annotations.SerializedName

import ru.artempugachev.popularmovies.model.Review

/**
 * Data class for review response
 */

data class ReviewResponse (
    @SerializedName("id")
    val id: String,

    @SerializedName("page")
    val page: Int,

    @SerializedName("results")
    val results: List<Review>
)
